### DISCUSSION TITLE
ARM benchmark against libjpeg-turbo/libpng

### DISCUSSION THREAD

**Discussion by Jeevhi** [nothings/stb#1362] [Q&A]
Hi,

stb_image is incredibly simple and impressive. I plan to use it in Android and iOS app and wondering about the benchmark especially on ARM processor, especially for compression, decompression, resize. 

I found one article but it's quite dated and I am certain that there are many improvements after that 

https://blog.gibson.sh/2015/03/23/comparing-performance-stb_image-vs-libjpeg-turbo-libpng-and-lodepng/

Any inputs please

**Comment by nothings** [comment]
Performance is not a primary goal for any of these libraries, and there probably haven't been any significant performance improvements since 2015.
